src/autocomplete/DuckDuckGoProvider.js

```diff
@@ -1,4 +1,21 @@
+/*
```

Comment:
Hmm, the autocomplete stuff will actually be (c) 2016 Aviral Dasgupta - we should probably not claim copyright for ourselves when that's not the case.